query/control/memory.go

```diff
@@ -28,6 +28,22 @@ type memoryManager struct {
 	unlimited bool
 }
 
+func (m *memoryManager) getUnusedMemoryBytes() int64 {
+	return atomic.LoadInt64(&m.unusedMemoryBytes)
+}
+
+func (m *memoryManager) getUsedMemoryBytes() int64 {
+	return atomic.LoadInt64(&m.unusedMemoryBytes)
```

Comment:
This method `getUsedMemoryBytes()` is returning `unusedMemoryBytes`

Reply:
thanks for catching that.

Certainly forgot to delete that. That's unused actually

Reply:
Now updated 👍